app/views/support_interface/data_exports/directory.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= render 'support_interface/performance/performance_navigation', title: 'Data directory' %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <% @export_types.each do |id, export| %>
+      <%# TODO: If first section, don’t add `app-section--with-top-border` class %>
```

Comment:
@JR-G Might you be able to help me solve this one… basically for the first section, don't want to show a top border.